spring-cloud-stream/src/main/java/org/springframework/cloud/stream/binding/BoundElementFactory.java

```diff
@@ -19,26 +19,35 @@
 import org.springframework.messaging.SubscribableChannel;
 
 /**
- * Defines methods to create/configure the {@link org.springframework.messaging.MessageChannel}s defined
- * in {@link org.springframework.cloud.stream.annotation.EnableBinding}.
+ * Defines methods to create/configure the bound elements defined by
+ * {@link org.springframework.cloud.stream.annotation.EnableBinding}.
+ *
+ * @author Marius Bogoevici
  * @author Ilayaperumal Gopinathan
  */
-public interface BindableChannelFactory {
+public interface BoundElementFactory {
```

Comment:
I wonder if `BindingTarget` might be a better name for `BoundElement` throughout? That is how we refer to this same "element" elsewhere in the codebase already, right?.